.github/workflows/phpunit-tests.yml

```diff
@@ -39,9 +39,14 @@ jobs:
                     - '8.1'
                     - '8.2'
                 multisite: [false, true]
+                wordpress: ['WordPress/WordPress']
```

Comment:
This is the value we're currently using in `.wp-env.json`, in order to introduce no change over the current behavior: https://github.com/WordPress/gutenberg/blob/f752350ee8da28110811c37edd4fd9b1f5cf4b61/.wp-env.json#L2

(This means that WP is installed from `trunk` rather than the zipball that's available for download at wordpress.org.)